<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #a4c847;</string>
   </property>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>837</width>
      <height>938</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>839</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frameGUI">
       <property name="minimumSize">
        <size>
         <width>839</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>839</width>
         <height>791</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Icons/AED_GUI_Background.png);
margin-left: -1px;
margin-right: -1px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QLabel" name="responseIndicator">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>340</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="helpIndicator">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="attachIndicator">
        <property name="geometry">
         <rect>
          <x>625</x>
          <y>420</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="contactIndicator">
        <property name="geometry">
         <rect>
          <x>635</x>
          <y>625</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="cprIndicator">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>640</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="shockIndicator">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>470</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>230</y>
          <width>280</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #d2d3d4;
border: 2px solid #8a8f9e;
border-radius: 10px;
margin: 0;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="shockCount">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
border: 0;</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>SHOCKS: 00</string>
         </property>
        </widget>
        <widget class="QLabel" name="elapsedTime">
         <property name="geometry">
          <rect>
           <x>225</x>
           <y>10</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
border: 0;</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
        <widget class="QLabel" name="textMsg">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
border: 0;</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>CHECK
RESPONSIVNESS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="cprDepth0cm" native="true">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>30</y>
           <width>30</width>
           <height>75</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black;
border-radius: 0;
border: 0;</string>
         </property>
        </widget>
        <widget class="QWidget" name="cprDepth5cm" native="true">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>106</y>
           <width>30</width>
           <height>15</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black;
border-radius: 0;
border: 0;</string>
         </property>
        </widget>
        <widget class="QWidget" name="cprDepth6cm" native="true">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>120</y>
           <width>30</width>
           <height>15</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black;
border-radius: 0;
border: 0;</string>
         </property>
        </widget>
        <widget class="QWidget" name="cprDepthMark5cm" native="true">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>105</y>
           <width>30</width>
           <height>2</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white;
border: 0;
border-radius: 0;</string>
         </property>
        </widget>
        <widget class="QWidget" name="cprDepthMark6cm" native="true">
         <property name="geometry">
          <rect>
           <x>248</x>
           <y>120</y>
           <width>30</width>
           <height>2</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white;
border: 0;
border-radius: 0;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>20</y>
          <width>121</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QProgressBar" name="batteryIndicator">
           <property name="styleSheet">
            <string notr="true">QProgressBar {
	border: 2px solid black;
	border-radius: 5px;
	padding: 1px;
}

QProgressBar::chunk {
     background-color: green;
     width: 3px;
     margin: 0.5px;
 }</string>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>5</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>5</width>
             <height>10</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: black;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="controlBtnBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #a4c847;
border: 0;</string>
       </property>
       <widget class="QToolButton" name="powerBtn">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>-18</y>
          <width>241</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Icons/power_btn_off.png</normaloff>
          <normalon>:/Icons/power_btn_on.png</normalon>:/Icons/power_btn_off.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="selftCheckIndicator">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-20</y>
          <width>241</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Icons/self_check_fail.png</normaloff>
          <normalon>:/Icons/self_check_success.png</normalon>
          <disabledoff>:/Icons/self_check_fail.png</disabledoff>
          <disabledon>:/Icons/self_check_success.png</disabledon>:/Icons/self_check_fail.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="patientInfoController">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>271</width>
      <height>915</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #808284;
border-left: 2px solid black;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>192</width>
       <height>55</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0;
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Icons/patient.png</normaloff>
          <disabledoff>:/Icons/patient.png</disabledoff>
          <disabledon>:/Icons/patient.png</disabledon>:/Icons/patient.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0;
color: black;</string>
        </property>
        <property name="text">
         <string>Patient Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="conditionSelector">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>170</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox::indicator {
    background-color: white;
    selection-background-color: white;
    color: black;
    selection-color: green;
}
</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>Healthy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ventricular Fibrillation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ventricular Tachycardia</string>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="cprPadsAction">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>640</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Attach CPR Pads</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
border-radius: 10px;
border: 2px solid #8a8f9e;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/Icons/attach_cpr_pads.png</normaloff>
       <normalon>:/Icons/detach_cpr_pads.png</normalon>:/Icons/attach_cpr_pads.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>125</width>
       <height>125</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="horizontalGroupBox_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>175</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">border: 0;</string>
        </property>
        <property name="text">
         <string>Number of Runs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numOfRunsSelector">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="shallowPushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>760</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Shallow Push</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deepPushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>810</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Deep Push</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>210</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0;
color: black;</string>
     </property>
     <property name="text">
      <string>Battery Units</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>212</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">border: 0;</string>
           </property>
           <property name="text">
            <string>Starting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="startingBatteryLevel">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">border: 0;</string>
           </property>
           <property name="text">
            <string>Idle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numOfRunsSelector_3">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">border: 0;</string>
           </property>
           <property name="text">
            <string>Per Shock</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numOfRunsSelector_2">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
